components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/AbstractAPIManager.java

```diff
@@ -490,8 +490,7 @@ public API getAPIbyUUID(String uuid, String organization) throws APIManagementEx
             Registry registry;
             if (organization != null && !MultitenantConstants.SUPER_TENANT_DOMAIN_NAME.equals
                     (organization)) {
-                int id = getTenantManager()
-                        .getTenantId(organization);
+                int id = APIUtil.getInternalOrganizationId(organization);
                 startTenantFlow(organization);
```

Comment:
getAPIbyUUID() is not used. APIProviderImpl has the overridden method getAPIbyUUID()